src/main/java/org/pesmypetcare/webservice/entity/medalmanager/UserMedalEntity.java

```diff
@@ -0,0 +1,68 @@
+package org.pesmypetcare.webservice.entity.medalmanager;
+
+import lombok.Data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Oriol Catalán
+ */
+@Data
+public class UserMedalEntity {
+    public static final String NAME = "name";
+    public static final String PROGRESS = "progress";
+    public static final String CURRENT_LEVEL = "currentLevel";
+    public static final String COMPLETED_LEVELS_DATE = "completedLevelsDate";
+    private String name;
+    private Double progress;
+    private Double currentLevel;
+    private ArrayList<String> completedLevelsDate;
+
+    public UserMedalEntity() { }
+
+    public UserMedalEntity(String name, Double progress, Double currentLevel, ArrayList<String> completedLevelsDate) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using implementation types like 'ArrayList'; use the interface instead](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5517612)